packages/components/src/FilterDateRangePicker/FilterDateRangePicker.spec.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useState } from "react"
 import { render, waitFor } from "@testing-library/react"
 import userEvent from "@testing-library/user-event"
 import { FilterButton } from "~components/FilterButton"
-import { DateRange } from "~types/DatePicker"
+import { DateRange } from "~types/date-controls"
```

Comment:
Should this type come from `DateRangePicker` or `CalendarRange`?

Reply:
Hmm... Yeah I really struggled with the location for the shared types. I wanted it in `DateRangePicker`, but that's currently a legacy version and I didn't want it to house non-legacy types (in case they deviate and it breaks `FilterDRP`).
But putting it in `FilterDRP` seemed wrong too, given it's meant to be somewhat of a wrapper only.

I didn't want to put it in `CalendarRange` because it doesn't actually use `DateRange` itself (well, it does, but it's in the spread props).

I put it in `date-controls/ReactDayPicker` along with `DateInterval` (exposed so consumers can use it to type against the disabled days abstraction; although now that I think about it, they can also just drill into the larger type so this probably isn't necessary), but open to changing it to wherever makes the most sense (I could move them all into Calendar maybe? Since DayPicker is the main reason for their existence).